samples/nextjs/jss-create.js

```diff
@@ -17,5 +20,91 @@ module.exports = function createJssProject(argv, nextSteps) {
 
   applyNameToProject(__dirname, argv.name, argv.hostName, 'JssNextWeb');
 
+  if (!argv.fetchWith || !argv.prerender) {
+    nextSteps.push([
+      `* Did you know you can customize the Next.js sample app using ${chalk.green('jss create')} parameters?`,
+      `*  ${chalk.green('--fetchWith {REST|GraphQL}')} : Specifies how Sitecore data (layout, dictionary) is fetched. Default is REST.`,
+      `*  ${chalk.green('--prerender {SSG|SSR}')} : Specifies the Next.js pre-rendering form for the optional catch-all route. Default is SSG.`,
+    ]);
+  }
+
+  setFetchWith(argv.fetchWith);
+  setPrerender(argv.prerender);
+
   return nextSteps;
 };
+
+/**
+ * Sets how Sitecore data (layout, dictionary) is fetched.
+ * @param {string} [fetchWith] {REST|GraphQL} Default is REST. 
+ */
+function setFetchWith(fetchWith) {
+  const defaultDsfFile = path.join(__dirname, 'src/lib/dictionary-service-factory.ts');
+  const graphqlDsfFile = path.join(__dirname, 'src/lib/dictionary-service-factory.graphql.ts');
+  const defaultLsfFile = path.join(__dirname, 'src/lib/layout-service-factory.ts');
+  const graphqlLsfFile = path.join(__dirname, 'src/lib/layout-service-factory.graphql.ts');
+  const FetchWith = {
```

Comment:
Any particular reason not to use actual enums? (https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) And I was thinking it would be exported in case people needed to do more customizations based on this value. But maybe I'm overthinking this whole scenario.